consensus/polybft/helpers_test.go

```diff
@@ -19,6 +19,18 @@ func createTestKey(t *testing.T) *wallet.Key {
 	return wallet.NewKey(wallet.GenerateAccount())
 }
 
+func createRandomTestKeys(t *testing.T, amount int) []*wallet.Key {
```

Comment:
Nitpick - maybe call this `amount` parameter - `numberOfKeys`.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1167/commits/a2201f5a6ab19aa537d88ba92ceac9806bdbca5a